test/internet/test-dns-ipv6.js

```diff
@@ -44,46 +40,49 @@ function checkWrap(req) {
 }
 
 TEST(function test_resolve6(done) {
-  var req = dns.resolve6('ipv6.google.com', function(err, ips) {
-    if (err) throw err;
+  const req = dns.resolve6('ipv6.google.com',
+    common.mustCall((err, ips) => {
```

Comment:
@edsadr I think you have `common.mustCall()` and everything below it indented by 3 spaces too much. I think it should be like this:

```js
const req = dns.resolve6('ipv6.google.com',
                         common.mustCall((err, ips) => {
                           assert.ifError(err);

                           assert.ok(ips.length > 0);

                           for (let i = 0; i < ips.length; i++)
                             assert.ok(isIPv6(ips[i]));

                           done();
                         }));

```